lib/circuitry/subscriber.rb

```diff
@@ -119,16 +119,18 @@ def poll(&block)
       end
 
       poller.poll(max_number_of_messages: batch_size, wait_time_seconds: wait_time, skip_delete: true) do |messages|
-        messages = parse_and_filter_messages(messages)
-        process_messages(messages, &block)
+        messages = message_parser(messages)
+        filtered_messages = filter_messages(messages)
+        delete_invalid_messages(messages, filtered_messages)
```

Comment:
I suggest using another word for `invalid` like `irrelevant` e.g. delete_irrelevant_messages_from_queue

Reply:
Replaced `invalid` with `excluded`. Been puzzling over whether we need to include `_from_queue` (or something like that) in the function name given the variable name within the function (`messages_to_delete_from_queue`). Conclusion: it's just hard to succinctly express both A. we're deleting the message from SQS, but B. the message is excluded by us. Any suggestions?

Reply:
good call @ada-ebling, i don't think we need to be redundant with the naming as with what you mentioned, the variable name within the method mentions it